.github/workflows/gradle.yml

```diff
@@ -141,6 +147,7 @@ jobs:
         generate-branches-badge: true
     - name: Commit Coverage Badge
       uses: stefanzweifel/git-auto-commit-action@v4
+      if: ${{ ! github.event.inputs.mavenFilesPreparation }}
```

Comment:
Disables the commit of the code coverage badge when running the Maven on-demand workflow.